util.c

```diff
@@ -282,6 +283,64 @@ int challenge_response(YK_KEY *yk, int slot,
 }
 
 int
+check_user_challenge_file(const char *chalresp_path, const struct passwd *user, FILE *debug_file)
+{
+  /*
+   * This function will look for users challenge files.
+   *
+   * Returns one of AUTH_FOUND, AUTH_NOT_FOUND, AUTH_ERROR
+   */
+  int len;
```

Comment:
len should probably be a size_t since it's assigned result of strlen(), passed to malloc() and passed to snprintf()